tests/congruence_tests/test_search.py

```diff
@@ -205,25 +235,26 @@ def test_simple_opt_vectors_search():
                 local_client, remote_client, searcher.filter_search_text, query_filter=query_filter
             )
         except AssertionError as e:
-            print(f"\nFailed with filter {query_filter}")
-            raise e
+            raise ValueError("Vector:", searcher.query_text, "Filter:", query_filter) from e
 
 
-def test_single_vector():
-    fixture_points = generate_fixtures(num=200, vectors_sizes=text_vector_size)
 
+def test_single_vector():
+    fixture_points = generate_fixtures(num=201, vectors_sizes=text_vector_size)
+    query, fixture_points = fixture_points[-1], fixture_points[:-1]
     searcher = TestSimpleSearcher()
+    searcher.query_text = np.array(query.vector).tolist()
 
     vectors_config = models.VectorParams(
         size=text_vector_size,
         distance=models.Distance.DOT,
     )
 
     local_client = init_local()
-    init_client(local_client, fixture_points, vectors_config=vectors_config)
+    init_client(local_client, fixture_points, vectors_config=vectors_config, init_debug_collection=True)
```

Comment:
were we meant to delete debug code?

Reply:
That's a collection to debug it in future. 
We can save EXACTLY the breaking vector and reproduce any kind of situation of stochastic tests rent offline

Reply:
I would like to keep it

Reply:
We're running tests with pytest -x, so tests won't run if there's a failure and collections will be persisted